mobile-app/lib/ui/views/learn/widgets/assignment_widget.dart

```diff
@@ -53,7 +54,14 @@ class Assignment extends StatelessWidget {
                 children: parser.parse(
                   label,
                   isSelectable: false,
-                  fontColor: value ? const Color(0xDEFFFFFF) : null,
+                  customStyles: value
+                      ? {
+                          '*:not(h1):not(h2):not(h3):not(h4):not(h5):not(h6)':
+                              Style(
+                            color: const Color(0xDEFFFFFF),
```

Comment:
We should change this to `FccColors.gray00`

Reply:
We should just call this color as white honestly 🤣